contrib/benchmarking_nd4j/src/main/java/com/example/NativeOps.java

```diff
@@ -0,0 +1,97 @@
+package com.example;
+
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.ops.impl.scalar.ScalarMultiplication;
+import org.nd4j.linalg.api.ops.impl.transforms.strict.Exp;
+import org.nd4j.linalg.factory.Nd4j;
+import org.openjdk.jmh.annotations.*;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class NativeOps {
+
+    @State(Scope.Thread)
+    public static class SetupState {
+        INDArray array = Nd4j.ones(1024, 1024);
+        INDArray arrayRow = Nd4j.linspace(1, 1024, 1024);
+        INDArray arrayColumn = Nd4j.linspace(1, 1024, 1024).reshape(1024,1);
+        INDArray array1 = Nd4j.linspace(1, 20480, 20480);
+        INDArray array2 = Nd4j.linspace(1, 20480, 20480);
+
+        INDArray array3 = Nd4j.ones(128, 256);
+        INDArray arrayRow3 = Nd4j.linspace(1, 256, 256);
+
+        INDArray arrayUnordered = Nd4j.ones(512, 512);
+        INDArray arrayOrderedC = Nd4j.zeros(512, 512,'c');
+        INDArray arrayOrderedF = Nd4j.zeros(512, 512, 'f');
+
+        {
+            float sum = (float) array.sumNumber().doubleValue();
+        }
```

Comment:
Don't really remember what this is for, I guess it would make sense to just get rid of it.

Reply:
Removed.